corehq/apps/hqwebapp/templatetags/hq_shared_tags.py

```diff
@@ -257,7 +257,7 @@ def can_use_restore_as(request):
     return (
         request.couch_user.can_edit_commcare_users() and
         has_privilege(request, privileges.DATA_CLEANUP) and
```

Comment:
i mean i understand why it's there, it's because with login as you can technically do a lot of data cleanup that you can't do normally on standard plan. but if we want to make it available for standard plan then we have to remove it...